### PR TITLE
[DEV-2128] Divide config height in a vertical & horizontal 

### PR DESCRIPTION
**Pull request by Atash3000**
## Briefly describe your changes
Divided chart height into two parts. Vertical charts can be manually updated by user.Horizontal charts is gets updated dynamically based on bar height, bar space etc..
Fixed: When switching from horizontal charts to vertical, the vertical chart used to inherit the size(height) from horizontal.
now it doesn't inherit. each chart has own height.
## Checklist before requesting a review
- [x] My pull request was branched from and targets the test branch
- [x] I have performed a self-review of my code
- [x] I have manually tested all packages affected (bonus points for automations)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have checked to ensure there aren't other open pull requests for the same change

## Did you test your feature in the following environments?
- [x] Standalone Component
- [ ] Standalone Full Editor
- [ ] CDC Internal Checks

### CODE REVIEW

File: packages/chart/src/components/LinearChart.tsx [hunk 1]
```diff
@@ -505,7 +502,7 @@ export default function LinearChart() {
                       {!config.runtime.yAxis.hideAxis && <Line from={props.axisFromPoint} to={props.axisToPoint} stroke='#333' />}
                     </Group>
                     <Group>
-                      <Text transform={`translate(${xMax / 2}, ${config.height - yMax + 20}) rotate(-${0})`} verticalAnchor='start' textAnchor={'middle'} stroke='#333'>
+                      <Text transform={`translate(${xMax / 2}, ${yMax + 20}) rotate(-${0})`} verticalAnchor='start' textAnchor={'middle'} stroke='red'>
```
Comment (adamdoe) [inline packages/chart/src/components/LinearChart.tsx:505]: @Atash3000 - Is the red stroke here intentional?

File: packages/chart/src/components/LinearChart.tsx [hunk 2]
```diff
@@ -28,7 +26,6 @@ export default function LinearChart() {
   let [width] = dimensions
   const { minValue, maxValue, existPositiveValue, isAllLine } = useReduceData(config, data)
   const [animatedChart, setAnimatedChart] = useState<boolean>(false)
-  const [animatedChartPlayed, setAnimatedChartPlayed] = useState<boolean>(false)
```
Comment (adamdoe) [inline packages/chart/src/components/LinearChart.tsx:31]: good

File: packages/chart/src/components/LinearChart.tsx [hunk 3]
```diff
@@ -16,7 +15,6 @@ import useIntersectionObserver from './useIntersectionObserver'
 import CoveBoxPlot from './BoxPlot'
 
 import ErrorBoundary from '@cdc/core/components/ErrorBoundary'
-import numberFromString from '@cdc/core/helpers/numberFromString'
```
Comment (adamdoe) [inline packages/chart/src/components/LinearChart.tsx:19]: good

File: packages/chart/src/components/LinearChart.tsx [hunk 4]
```diff
@@ -6,7 +6,6 @@ import { Line } from '@visx/shape'
 import { Text } from '@visx/text'
 import { scaleLinear, scalePoint, scaleBand } from '@visx/scale'
 import { AxisLeft, AxisBottom, AxisRight, AxisTop } from '@visx/axis'
-import * as d3 from 'd3-array'
```
Comment (adamdoe) [inline packages/chart/src/components/LinearChart.tsx:9]: good

File: packages/chart/src/components/LinearChart.tsx (hunk 1 above)
Comment (Atash3000) [reply packages/chart/src/components/LinearChart.tsx:505]: Sorry i forgot to remove.

**Pull request merged**